stylesheets/_modules.scss

```diff
@@ -5516,7 +5516,7 @@ button.module-image__border-overlay:focus {
 
 .module-sticker-picker {
   @extend %module-composition-popper;
-  height: 400px;
+  height: 350px;
```

Comment:
We're reducing these values globally, and they're already a little small, especially when the window is big. We can merge this if we get dynamic sizing in these pickers you've updated. Small when the window is small, so they fit. Bigger when the window is bigger.